client/state/plugins/installed/selectors-ts.ts

```diff
@@ -65,6 +72,34 @@ export const getAllPluginsIndexedByPluginSlug = createSelector(
 	]
 ) as { ( state: AppState ): { [ pluginSlug: string ]: Plugin } };
 
+const _filters: { [ key in PluginFilter ]: ( plugin: Plugin ) => boolean } = {
```

Comment:
I know this is duplicated with `selectors.js`. My current plan is to eventually duplicate everything here until `selectors-ts.ts` is equivalent to `selectors.js`, at which point I will delete `selectors.js` and rename `selectors-ts.ts` to `selectors.ts`.